spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -73,21 +73,21 @@ install_kind_release() {
 	elif [[ "$OSTYPE" == "win32" ]]; then
         KIND_BINARY_URL="https://github.com/kubernetes-sigs/kind/releases/download/${VERSION}/kind-windows-amd64"
 	else
-        echo "Uknown OS, using linux binary"
+        echo "Unknown OS, using linux binary"
 	fi
     wget -O "${KIND}" "${KIND_BINARY_URL}"
     chmod +x "${KIND}"
 }
 
 main() {
     # get kind
-    install_latest_kind
+    install_kind_release
 
     # create a cluster
     cd $CURRENT_DIR
 
     #TODO what happens if cluster is already there????
-    "${KIND}" create cluster --config=kind-config.yaml --loglevel=debug
```

Comment:
`-v=2147483647` is like `log::trace` in my understanding, sort of like saying "max level output possible"